validate_form.js

```diff
@@ -65,3 +65,31 @@ form.addEventListener('submit', (e) => {
   }
   updateInput();
 });
+
+const contactBtn = document.querySelector('.contact-btn');
+
+function storeData() {
+  const saveName = userName.value;
+  const saveMail = userMail.value;
+  const saveComment = commentText.value;
+
+  const dataobject = { Name: saveName, Mail: saveMail, Comment: saveComment };
```

Comment:
The form data for all inputs is be stored in a single object, saved to a single local storage registry 🙌 Well done!